cfgmgr/teammgr.cpp

```diff
@@ -658,9 +658,6 @@ bool TeamMgr::removeLag(const string &alias)
     string res;
     pid_t pid;
 
-    cmd << TEAMD_CMD << " -k -t " << shellquote(alias);
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
-
```

Comment:
FWIW, can I suggest to the following:

```
bool TeamMgr::removeLag(const string &alias, bool assertSuccess)
{
...
if (asserSuccess) {
    EXEC_WITH_ERROR_THROW(cmd.str(), res);
} else {
    exec( cmd.str(), res );
}
   ...
}
```
In the case when the `addLag()` fails, `removeLag()` will be called with `assertSuccess=false`, otherwise the code path stays the same. IMO this would address the issue while still keeping the `addLag()` and `removeLag()` symmetrical... my 2cents.